faq.md

```diff
@@ -107,9 +107,9 @@ as handwritten code. If you experience any weirdness, file a bug!
 
 <!-- References -->
 
-[`@Binds`]: http://google.github.io/dagger/api/latest/dagger/Binds.html
-[`@Provides`]: http://google.github.io/dagger/api/latest/dagger/Provides.html
-[`dagger.producers.monitoring`]: http://google.github.io/dagger/api/latest/dagger/producers/monitoring/package-summary.html
+[`@Binds`]: api/latest/dagger/Binds.html
```

Comment:
Sure. I think you can even skip the scheme, which might make local testing easier.

`//google.github.io/dagger/api/latest/dagger/Binds.html` is a real relative URL that uses the same scheme as the current URL. (I hope it works.)